### PR TITLE
Fix CI as per failure in main (action.yml)

### PR DESCRIPTION
**Pull request by paulo-ferraz-oliveira**
This hopefully fixes the `action.yml` failure we're seeing in `main`. It is possible the changes were committed while the license was being applied, though it's uncommon this happens.

### CODE REVIEW

**Review by wojtekmach — APPROVED** [review]

**Comment by paulo-ferraz-oliveira** [comment]
Hm... still failing.

File: package-lock.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "requires": true,
   "packages": {
     "": {
+      "name": "setup-beam",
```
Comment (paulo-ferraz-oliveira) [inline package-lock.json:8]: Not really sure where this stuff comes from. 🤷‍♂️

Comment (starbelly) [reply package-lock.json:8]: wdym?

Comment (paulo-ferraz-oliveira) [reply package-lock.json:8]: @starbelly, is this Ok to merge, still? Thanks much.

Comment (paulo-ferraz-oliveira) [reply package-lock.json:7]: > Not really sure where this stuff comes from. 🤷‍♂️

It comes from Node.js 14 with `npm install`.

**Comment by paulo-ferraz-oliveira** [comment]
@starbelly, is this Ok to merge, still? Thanks much.

**Comment by paulo-ferraz-oliveira** [comment]
Nothing really critical change, so I'm going to go ahead and squash-merge this.

**Pull request merged**